tests/system/providers/amazon/aws/example_s3.py

```diff
@@ -200,7 +202,18 @@ def check_fn(files: list) -> bool:
         check_fn=check_fn,
         deferrable=True,
     )
-    # [END howto_sensor_s3_key_function]
+    # [END howto_sensor_s3_key_function_deferrable]
+
+    # [START howto_sensor_s3_key_regex_deferrable]
```

Comment:
Thanks for adding it but for now we cannot have deferrable operators in system tests, it is not compatible yet. Though, I would leave the non deferrable version below, it can be super useful for users!

Reply:
Looks like we already have a few examples of deferrable operators here. Should I remove them? or somehow disable them? Otherwise, we might not be able to include them in the doc

Reply:
yeah lets remove them from system tests

Reply:
What do we mean by they are not compatible yet? IMO, I would prefer keeping these system tests as they also act as a reference example on the usage of operators.